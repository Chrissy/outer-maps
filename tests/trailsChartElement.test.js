import React from "react";
import TrailsChartElement from "../components/trailsChartElement.js";
import renderer from "react-test-renderer";

const props = {
  name: "Fun Trail",
  url: "/trails/60",
  distance: 60,
  id: 1001
};

it("renders", () => {
  const tree = renderer.create(<TrailsChartElement {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
