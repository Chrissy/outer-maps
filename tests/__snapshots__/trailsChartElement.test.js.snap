// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
.emotion-4 {
  display: grid;
  grid-template-columns: 1fr 3em;
  font-size: 0.8em;
  line-height: 1.1em;
  cursor: pointer;
}

.emotion-4:nth-of-type(even) {
  background-color: #E9E9E9;
}

.emotion-0 {
  padding: 0.5em;
  color: #494949;
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.emotion-2 {
  padding: 0.5em;
  color: #A7A7A7;
  border-left: 1px solid #fff;
  text-align: center;
}

<div
  className="emotion-4 emotion-5"
>
  <div
    className="emotion-0 emotion-1"
  >
    Fun Trail
  </div>
  <div
    className="emotion-2 emotion-3"
  >
    0
  </div>
</div>
`;
